Doc/library/json.rst

```diff
@@ -678,40 +678,43 @@ when serializing instances of "exotic" numerical types such as
 
 
 .. _json-commandline:
-.. program:: json.tool
+.. program:: json
 
-Command Line Interface
+Command-Line Interface
```

Comment:
```suggestion
Command-line Interface
```

Reply:
I went with the "Command-Line Interface" spelling because I saw that one used 7 times in the documentation and "Command-line interface" used 2 times. I didn't see the spelling "Command-line Interface" used anywhere.

Reply:
Ahh, sorry, should have been lower-case 'i'. The style guide [recommends](https://devguide.python.org/documentation/style-guide/#capitalization) sentence case for titles.